src/alembic/versions/0016_add_fund_group.py

```diff
@@ -0,0 +1,30 @@
+"""add fund group column to fund table
+
+Revision ID: 0008
```

Comment:
These values don't seem to be correct. should be accordingly Revision ID:0016 and Revises:0015. Create Date: 2021-02-14 ....